### ISSUE TITLE
Performance on ScanNet

### ISSUE THREAD

**Issue by lailvlong** [Na-Z/attMPTI#8]
Thanks for your opening source the nice project. 
I run attMPTI on the two dataset using the hyper-parameters according to the paper. I can get the reported performance of the paper on S3DIS but fail on ScanNet. Concretly, in 2-way-1shot task on both split-0 and split-1 of ScanNet , i set n_prototypes=100, knn_neighbor=200, Gausian_sigma=5. The meanIoU i get are 39.5% and 37.6% on split-0 and split-1 respectively, which are about 3% dropped from the paper. I have also tried Gausian_sigma=1 but the result changes slightly.
Do you have these problems  when you conducting the experiments on ScanNet? Or do you know how i can fix these problems?

**Comment by Na-Z** [comment]
Hi, 

The different performance might be due to two reasons: 1) insufficient pre-training; 2) sample bias when randomly sampling testing set.

Since ScanNet contains more examples than S3DIS, we did a longer pre-training on ScanNet (150 epochs).  
You may want to try: 1) pre-train longer; 2) re-generate test set.

I can also send you my testing set on 2-way 1-shot setting if needed.

**Comment by lailvlong** [comment]
Thank you! I will have a try.

> Hi,
> 
> The different performance might be due to two reasons: 1) insufficient pre-training; 2) sample bias when randomly sampling testing set.
> 
> Since ScanNet contains more examples than S3DIS, we did a longer pre-training on ScanNet (150 epochs).
> You may want to try: 1) pre-train longer; 2) re-generate test set.
> 
> I can also send you my testing set on 2-way 1-shot setting if needed.

**Comment by lailvlong** [comment]
> Hi,
> 
> The different performance might be due to two reasons: 1) insufficient pre-training; 2) sample bias when randomly sampling testing set.
> 
> Since ScanNet contains more examples than S3DIS, we did a longer pre-training on ScanNet (150 epochs).
> You may want to try: 1) pre-train longer; 2) re-generate test set.
> 
> I can also send you my testing set on 2-way 1-shot setting if needed.

**Comment by lailvlong** [comment]
Hello! I run attMPTI on the S3DIS-S1-N2-K5 task with the paper's hyper parameters. The mIoU is about 61%, which is 6% lower than that in the paper. I have tried longer pre-training and regenerating the test set, but the improvement is inapparent. Is that the S3DIS-S1-N2-K5 task needs other particular tricks?

**Comment by Na-Z** [comment]
> Hello! I run attMPTI on the S3DIS-S1-N2-K5 task with the paper's hyper parameters. The mIoU is about 61%, which is 6% lower than that in the paper. I have tried longer pre-training and regenerating the test set, but the improvement is inapparent. Is that the S3DIS-S1-N2-K5 task needs other particular tricks?

Hi, we didn't use specific trick. When training S3DIS-S1-N2-K5, the setting is the same as that of S3DIS-S0-N2-K5 (i.e., k=200, n=150, sigma=1). Note that we trained 20000 iterations.

**Comment by 1170801121** [comment]
Hi, it seems you use different parameters for different dataset and few-shot setting that you did not mention in the paper. And you only provide one set of parameters for S3DIS-N2-K1 in the codebase.  Can you provide your parameters on other settings (i.e., S3DIS-N2-K5, Scannet-N2-K1, Scannet-N2-K5)? Thank you very much!
